admin/app/components/target-profiles/create-target-profile-form.hbs

```diff
@@ -1,100 +1,102 @@
-      <label class="form-field__label" for="skillsList">Fichier JSON Pix Editor
-        <abbr title="obligatoire" class="mandatory-mark">*</abbr>
-        :
-      </label>
-      <Input
-        class={{if @isFileInvalid "form-control is-invalid" "form-control"}}
-        required="true"
-        {{on "change" @onLoadFile}}
-        @type="file"
-        id="skillsList"
-      />
-      {{#if @isFileInvalid}}
-        <p class="form-field__error">Le fichier Pix Editor n'est pas au bon format.</p>
-      {{/if}}
-    </div>
+      <div class="form-field form-group">
+        <PixInput
+          @id="organizationId"
+          @label="Identifiant de l'organisation de référence : "
+          @errorMessage=""
+          required
+          placeholder="7777"
+          {{on "change" this.updateOwnerOrganizationId}}
+        />
+      </div>
+    </Card>
 
-    <div class="form-field form-group">
-      <label class="form-field__label" for="organizationId">Identifiant de l'organisation de référence : </label>
-      <Input
-        class="form-control"
-        id="organizationId"
-        placeholder="7777"
-        @value={{@targetProfile.ownerOrganizationId}}
-      />
-    </div>
+    <Card class="create-target-profile__card" @title="2. Sélection des sujets">
+      <div class="create-target-profile__inline-layout">
+        <div class="file-upload-group">
+          <label class="file-upload-group__label {{if @isFileInvalid "s-invalid"}}">Importer un profil cible .JSON
+            <Input class="file-upload-group__input" {{on "change" @onLoadFile}} @type="file" id="skillsList" />
+          </label>
+          {{#if @isFileInvalid}}
+            <p class="form-field__error">Le fichier Pix Editor n'est pas au bon format.</p>
+          {{/if}}
+        </div>
+      </div>
+    </Card>
 
-    <div class="form-field form-group">
-      <label class="form-field__label" for="imageUrl">Lien de l'image du profil cible :</label>
-      <Input class="form-control" id="imageUrl" @value={{@targetProfile.imageUrl}} />
-      <p class="form__instructions create-target-profile__instructions">L'url à saisir doit être celle d'OVH. Veuillez
-        vous rapprocher des équipes tech et produit pour la réalisation de celle-ci.</p>
-    </div>
-
-    <div class="form-field form-group">
-      <label class="form-field__label" for="description">Description : </label>
-      <PixTextarea
-        class="form-control"
-        id="description"
-        @maxlength="500"
-        rows="4"
-        @value={{@targetProfile.description}}
+    <Card class="create-target-profile__card" @title="3. Personnalisation">
+      <PixInput
+        @id="imageUrl"
+        class="input-url"
+        @label="Lien de l'image du profil cible : "
+        @information="L'url à saisir doit être celle d'OVH. Veuillez
+          vous rapprocher des équipes tech et produit pour la réalisation de celle-ci."
+        {{on "change" this.updateImageUrl}}
```

Comment:
**question** : pourquoi parfois on a `@value={{@targetProfile.description}}` et parfois on passe par `{{on "change" this.updateImageUrl}}` ?

Reply:
Yes c'est exactement ça (on a écrit une ADR sur le changement des Inputs d'ember aux inputs natifs : https://github.com/1024pix/pix-ui/blob/dev/docs/adr/0001-utiliser-les-balises-html-natives.md ).
Le `@value` ne permet plus le double binding sur les PixInputs (c'est depuis la version  de Pix UI 11.1.0). 

Par contre j'ai appris récemment que le `value=` pouvait toujours servir pour mettre le champ vide au besoin (exemple dans `admin/app/components/organizations/team-actions-section.hbs` où on a un PixInput avec un `value={{@userEmailToAdd}}` qui est remis à null `this.userEmailToAdd = null;` dans `admin/app/controllers/authenticated/organizations/get/team.js`).

La personne qui en parlera le mieux c'est @yschoueri  au besoin :D